app/assets/javascripts/Rickshaw.Graph.DragZoom.js

```diff
@@ -0,0 +1,131 @@
+Rickshaw.namespace('Rickshaw.Graph.DragZoom');
```

Comment:
@stuartnelson3 I don't think so, but this would be an internal matter, best discussed maybe with mobile teams :) In general, I'd be surprised if there were many orgs out there that allowed sufficiently privileged VPN on iPads to use monitoring, but who knows...